### DISCUSSION TITLE
Getting Trajectory Data after Video Run

### DISCUSSION THREAD

**Discussion by sadsasuke** [stella-cv/stella_vslam#361] [Q&A] [answered]
Hi,
How to get the coordinates and poses after the video slam run has ended. It save a  .msg file but I also cant read it. I want to be able to get trajectory and poses file and plot them.
Thanks.

**Comment by ymd-stella** [comment, marked as answer]
Use `eval-log-dir`.

https://github.com/stella-cv/stella_vslam/blob/de7781b1e268afcf6d313bcf881bf3678bc8835a/example/run_video_slam.cc#L199
